docs/en/table_design/table_types/Comparison_of_abilities_of_various_tables.md

```diff
@@ -0,0 +1,43 @@
ported (Data rows with the same Aggregate Key value will be aggregated)</td><td>Supported (Data rows with the same Unique Key value will be updated) </td></tr><tr><td><strong>UPDATE through data loading</strong> </td><td>Not supported </td><td>Supported (This can be achieved by using replace as the aggregate function) </td><td>Supported (The Unique Key table itself can be considered as an Aggregate table using the replace aggregatie function) </td></tr><tr><td><strong>DELETE through data loading</strong> </td><td>Supported. <a href='../../loading/Load_to_Primary_Key_tables.md'>Configure <code>__op=1</code> in the load job to achieve DELETE</a>. </td><td colspan="3">Not supported </td></tr><tr><td><strong>Integrity of</strong> <strong>data column values to be loaded</strong> </td><td>However, if partial column update (<code>partial_update</code>) is enabled, or a column has a default value, not all column values need to be loaded.</td><td>By default, all column values need to be loaded.However, if a column has a default value, not all column values need to be loaded. </td><td>By default, all column values need to be loaded. However, Aggregate tables can achieve partial column update by specifying the aggregate functions for value columns as REPLACE_IF_NOT_NULL. For details, see <a href='../../sql-reference/sql-statements/data-definition/CREATE_TABLE.md#column_definition'>aggr_type</a> Also, if a column has a default value, not all column values need to be loaded.</td><td>By default, all column values need to be loaded.However, if a column has a default value, not all column values need to be loaded.</td></tr><tr><td><strong>DML</strong> <strong><a href='../../sql-reference/sql-statements/data-manipulation/INSERT.md'>INSERT</a></strong></td><td colspan="4">Supported</td></tr><tr><td><strong>DML</strong> <strong><a href='../../sql-reference/sql-statements/data-manipulation/UPDATE.md'>UPDATE</a></strong> </td><td><ul><li>Key columns as filter conditions: Supported</li><li>Value columns as filter conditions: Supported </li></ul></td><td colspan="3">Not supported</td></tr><tr><td><strong>DML</strong> <strong><a href='../../sql-reference/sql-statements/data-manipulation/DELETE.md'>DELETE</a></strong></td><td><ul><li>Key columns as filter conditions: Supported</li><li>Value columns as filter conditions: Supported</li></ul></td><td><ul><li>Key columns as filter conditions: Supported</li><li>Value columns as filter conditions: Supported</li></ul>Note that only simple filter conditions based on key or value columns themselves are supported, such as =, &lt;, &gt;. Complex filter conditions, such as functions or subqueries, are not supported.</td><td colspan="2">Key columns as filter conditions: Supported. Note that only simple filter conditions based on key columns themselves are supported, such as =, &lt;, &gt;. Complex filter conditions, such as functions or subqueries, are not supported.Value columns as filter conditions: Not supported. </td></tr></tbody>
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/61d43638-6ce5-44cf-944c-3c32117bb081)
The content in this cell should be made as a bulleted list.